CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # 5.8.3 (in progress)
```

Comment:
Since this is the first "breaking change" can you set the in-progress version number to 6.0.0?  And I think it'd be nice to add a `##### Breaking Changes` section.

Reply:
Actually PR #1725 did this so just need to rebase with master branch.